pytket/extensions/azure/backends/azure.py

```diff
@@ -78,6 +76,26 @@ def _get_workspace(
 }
 
 
+_ADDITIONAL_GATES = {
```

Comment:
We should now be able to improve the default compilation passes for quantinuum devices.

Reply:
I would do this in a separate PR?